packages/rulesets/src/spectral/functions/avoid-additional-properties.ts

```diff
@@ -0,0 +1,18 @@
+const avoidAdditionalProperties = (schema: any, _opts: any, paths: any) => {
+  if (schema === null) {
+    return []
+  }
+  const path = paths.path || {}
+  if ("tags" === path[path.length - 1] && "additionalProperties" in schema) {
+    return
+  } else if ("tags" !== path[path.length - 1] && "additionalProperties" in schema) {
+    return [
+      {
+        message: "The use of additionalProperties is not allowed except for user defined tags on tracked resources.",
```

Comment:
>defined tags on tracked resources [](http://example.com/codeflow?start=81&length=33)

and for some system types like ManagedServiceIdentity and PrivateEndpointConnection

Reply:
Removed this file